android/res/values-zh-rTW/strings.xml

```diff
@@ -1161,6 +1161,7 @@
 	<string name="type.leisure.fitness_centre">健身中心</string>
 	<string name="type.leisure.fitness_station">健身房</string>
 	<string name="type.leisure.garden">花園</string>
+	<string name="type.leisure.garden.residential">居民区花园</string>
```

Comment:
Looks like this is in `zh-TW`, it is Traditional Chinese, and what I say only can apply to zh-Hans/zh-CN(or sometimes only zh).

Maybe I need to ask for friends in Taiwan community.

Reply:
Don't worry, i'll just remove it from zh-Hant & use the garden translation 👍